# This is a message with the raw data from an IMU
# angular_velocity is expected in deg/s
# magnetic_field is expected in Gs
# linear_acceleration is expected in g's

int16[3] angular_velocity
int16[3] magnetic_field
int16[3] linear_acceleration
